HISTORY.md

```diff
@@ -0,0 +1,306 @@
+# History
+
+## 2.3.2
```

Comment:
this needs the normal header from the changelig generator. then it will append the history.md to changelog.md.

Reply:
do you have an example? i cannot follow here :(